core/modules/user/user.module

```diff
@@ -1411,26 +1415,28 @@ function user_menu() {
     'menu_name' => 'user-menu',
   );
 
+  $user_login_page_settings = user_login_page_settings();
+  $tabs = $user_login_page_settings == 'tabs';
   $items['user/login'] = array(
     'title' => 'Log in',
     'page callback' => 'backdrop_get_form',
     'page arguments' => array('user_login'),
     'access callback' => 'user_is_anonymous',
-    'type' => MENU_DEFAULT_LOCAL_TASK,
+    'type' => ($tabs ? MENU_DEFAULT_LOCAL_TASK : MENU_CALLBACK),
```

Comment:
I still find this syntax irritating (wrapped in parenthesis), making "type" a bool (?).

```
'type' => $tabs ? MENU_DEFAULT_LOCAL_TASK : MENU_CALLBACK,
```
Same syntax is used twice below.

The only purpose of variable `$tabs` seems to be to distinguish between MENU_DEFAULT_LOCAL_TASK and MENU_CALLBACK - why not making a var $item_type instead - in a more readable way?

Reply:
Done